addon/components/item-picker/template.hbs

```diff
@@ -13,6 +13,10 @@
     </div>
   {{/if}}
 
+  {{#if showMessage}}
+    <div class="alert alert-warning" role="alert">{{currentMessage}}</div>
```

Comment:
I think we want to use the `state` to decide if this is `alert-warning` or `alert-danger` (for errors)